net.mancubus.SLADE.yaml

```diff
@@ -171,21 +162,17 @@ modules:
     sha256: aadb5597fb95ea5f7d336999e3c68ad32aa6357027aed7d4d9ff0ac75f2988e6
 
 - name : acc
-  buildsystem: simple
+  buildsystem: cmake-ninja
+  no-make-install: true
   build-commands:
-  - make
-  - install -Dm 774 acc /app/bin/acc
-  - install -Dm 774 acc.sh /app/bin/acc.sh
-  - install -Dm 644 zcommon.acs /app/bin/acc-include/zcommon.acs
-  - install -Dm 644 zdefs.acs /app/bin/acc-include/zdefs.acs
-  - install -Dm 644 zspecial.acs /app/bin/acc-include/zspecial.acs
-  - install -Dm 644 zwvars.acs /app/bin/acc-include/zwvars.acs
+  - install -Dm744 acc /app/bin/acc
+  - install -Dm744 acc.sh /app/bin/acc.sh
```

Comment:
@Calinou Question, for you, should we include the z*.acs headers? I forgot to add them so I'm quickly updating the script to include them again, but I never really thought about that. Does it make sense to have them?

Reply:
I don't know. If in doubt, include them anyway as to not break anything.